open_schools_platform/common/constants.py

```diff
@@ -7,39 +6,37 @@
 
 
 class CommonConstants:
-    GOOGLE_API_KEY = os.environ.get("GOOGLE_API_KEY")
-    GOOGLE_MAPS_API_KEY = os.environ.get("GOOGLE_MAPS_API_KEY")
+    GOOGLE_API_KEY = env("GOOGLE_API_KEY")
+    GOOGLE_MAPS_API_KEY = env("GOOGLE_MAPS_API_KEY")
     FIREBASE_URL_TO_GET_SESSION = r"https://www.googleapis.com/identitytoolkit" \
                                   r"/v3/relyingparty/sendVerificationCode?key="
     FIREBASE_URL_TO_CHECK_OTP = r"https://www.googleapis.com/identitytoolkit/v3" \
                                 r"/relyingparty/verifyPhoneNumber?key="
     FCM_URL_TO_VALIDATE_NOTIFICATIONS_TOKEN = "https://fcm.googleapis.com/fcm/send"
     SMS_PROVIDER_URL = r"https://sms.ru/sms/send"
-    SMS_API_KEY = os.environ.get("SMS_API_KEY")
-    FCM_SERVER_KEY = os.environ.get("FCM_SERVER_KEY")
-    SCHOOLS_AI_URL = os.environ.get("SCHOOLS_AI_URL")
+    SMS_API_KEY = env("SMS_API_KEY", default=None)
+    FCM_SERVER_KEY = env("FCM_SERVER_KEY", default=None)
```

Comment:
without this key we can't store media files,
I think there should be e runtime error, so we should delete default=None